search.java

```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.io.File;
+
+public class search {
+
+    public static void main(String[] args) throws FileNotFoundException {
```

Comment:
Could use a try/catch for a specific outprint message instead of "throws FileNotFoundException" here.